src/main/java/com/example/ride/serviceImpl/UserRegistrationServiceImpl.java

```diff
@@ -117,7 +110,68 @@ public GenericWebServiceResponse userlogin(UserRegistrationRequest request) {
 			throw new RuntimeException(e.getMessage());
 		} catch (Exception e) {
 			throw new GenericException("An error occurred while login the user " + request.getPhoneNumber());
+		}
+	}
+
+	@Override
+	public GenericWebServiceResponse getAllUserDetails() {
```

Comment:
Here i am not getting the response in form of JSON , I am getting in form of String.
<img width="790" alt="image" src="https://github.com/user-attachments/assets/c9a0f622-c324-4599-a163-ad6e0a01a5c0" />